### ISSUE TITLE
Wrench - Check Style - should scroll to the first issue

### ISSUE THREAD

**Issue by zrzka** [omz/Pythonista-Issues#349]
Let's say you have a long script. First half is visible, second one is not visible. First half has not style issue, second half has. Issue Wrench - Check Style. Wrench menu disappears and that's all. Nothing happens in terms of UI. IMO it should scroll to the line with first line issue. My workflow is pretty non standard now:

* Wrench - Check Style
* No Issues Found HUD -> okay, it's good
* Nothing happens -> scrolling up & down to find what's wrong

**Comment by dgelessus** [comment]
FYI, the file outline (the list that you get when tapping the filename, which lists all functions and classes) also includes all issues highlighted in the editor. You can use that to quickly jump to each issue. But I agree, there should be some kind of notification when the style checker did find issues.

**Comment by zrzka** [comment]
Yup, I don't know why, but I always forget it because there's no any kind of visual notification (HUD, badge with number of issues around the outline icon).

As a shortcut guy, I can imagine that Ole can provide shortcuts for go to next / previous issue / annotation. I know about Cmd-L, up & down & enter to jump to specific line. But it's kinda slow if you have lot of annotations / issues, you can't run Check Style / Analyze with keyboard shortcut again to clear them after fix, ...
